aws/http_request_retry_test.go

```diff
@@ -37,15 +32,13 @@ func TestRequestCancelRetry(t *testing.T) {
 	s.Handlers.UnmarshalError.Clear()
 	s.Handlers.Send.PushFront(func(r *aws.Request) {
 		reqNum++
-		r.Error = errors.New("net/http: request canceled")
 	})
 	out := &testData{}
 	r := s.NewRequest(&aws.Operation{Name: "Operation"}, nil, out)
 	r.HTTPRequest.Cancel = c
```

Comment:
Since this is v2 should we change this particular test to not use the deprecated cancel behavior while we are making modifications?